dali/operators/reader/numpy_reader_op.cc

```diff
@@ -77,7 +79,7 @@ static void SliceHelper(Tensor<CPUBackend> &output, const Tensor<CPUBackend> &in
   ), DALI_FAIL(make_string("Unsupported number of dimensions: ", ndim)););  // NOLINT
 }
 
-static void SlicePermuteHelper(Tensor<CPUBackend> &output, const Tensor<CPUBackend> &input,
+static void SlicePermuteHelper(SampleView<CPUBackend> output, ConstSampleView<CPUBackend> input,
```

Comment:
Now that you're at it...
```suggestion
static void SliceTransposeHelper(SampleView<CPUBackend> output, ConstSampleView<CPUBackend> input,
```
We use "Tranpose" in the other function and it is a better name for a total reversal of axes.

Reply:
I can see the name comes from the kernels used. I won't be applying as it seems out of scope to discuss the naming of those functions in this PR.

Reply:
Whatever.